server/routes/index.ts

```diff
@@ -20,6 +23,21 @@ router.get('/health', async (req, res) => {
   res.send('OK').end()
 })
 
+router.get('/page',
+  limiter(),
+  async (req, res) => {
+    try {
+      if (!req.query.url) {
+        throw new Error('URL query param is not specified')
+      }
+      const { data } = await axiosBase.get(req.query.url as string)
```

Comment:
provide substack specific api, not generalized proxy - we never intended to do that and we need to prevent misuse and abuse

Reply:
Okay

Reply:
Done

Reply:
still here